src/main/scala/mesosphere/marathon/core/task/bus/MarathonTaskStatus.scala

```diff
@@ -7,9 +7,11 @@ sealed trait MarathonTaskStatus {
   def mesosStatus: Option[TaskStatus]
 }
 
+// FIXME (merge): Merge MarathonTaskStatus and MesosTaskStatus
```

Comment:
Perhaps it makes sense to have MesosTaskStatus? I see a couple of references. What do you think?